### DISCUSSION TITLE
Time based filters

### DISCUSSION THREAD

**Discussion by kisst** [AdguardTeam/AdGuardHome#4360] [Ideas]
To set digital life balance, and help focus while working I want to define which services, url's to block at which times.
Work e-mail, after working hours no-no, fun sites at work time also no-no, but the other way around is OK.
I'm currently doing this this with config file swapping based on CRON and maintain two version of the conf, but would be nice to have an integrated feature for it.

**Comment by ainar-g** [comment]
This seems to be feature request #951.

**Comment by Aikatsui** [comment]
And / Or https://github.com/AdguardTeam/AdGuardHome/issues/1203

**Comment by HWiese1980** [comment]
Am I right to assume this has not been implemented yet?
